e2e/storagepool-policy/openebs-account.yaml

```diff
@@ -0,0 +1,51 @@
+# Define the Service Account
+# Define the RBAC rules for the Service Account
+# Launch the maya-apiserver ( deployment )
+# Launch the maya-storagemanager ( deameon set )
+
+# Create Maya Service Account 
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: openebs-maya-operator
+  namespace: default
+---
+# Define Role that allows operations on K8s pods/deployments
+#  in "default" namespace
+# TODO : change to new namespace, for isolated data network
+# TODO : the rules should be updated with required group/resources/verb
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  namespace: default
+  name: openebs-maya-operator
+rules:
+- apiGroups: ["*"]
+  resources: ["services","pods","deployments", "events"]
+  verbs: ["*"]
+- apiGroups: ["*"]
+  resources: ["persistentvolumes","persistentvolumeclaims"]
+  verbs: ["*"]
+- apiGroups: ["storage.k8s.io"]
+  resources: ["storageclasses"]
+  verbs: ["*"]
+---
+# Bind the Service Account with the Role Privileges.
+# TODO: Check if default account also needs to be there
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: openebs-maya-operator
+  namespace: default
+subjects:
+- kind: ServiceAccount
+  name: openebs-maya-operator
+  namespace: default
+- kind: User
+  name: system:serviceaccount:default:default
+  apiGroup: rbac.authorization.k8s.io
+roleRef:
+  kind: ClusterRole
+  name: openebs-maya-operator
+  apiGroup: rbac.authorization.k8s.io
+
```

Comment:
Just a suggestion, if the openebs-account.yaml and Dockerfile aren't going to change much can they be moved to a common location, like GNUMakefile?

Reply:
Yesh they will be moved to a common location. Finally they will be referred from openebs/openebs location.